editor/src/clj/editor/localization.clj

```diff
@@ -318,8 +319,9 @@
   ([object ^Localization localization pattern]
    {:pre [(localization? localization)]}
    (apply-localization object (impl-format @localization pattern))
-   (when (message-pattern? pattern)
-     (send-without-thread-binding-reset javafx-executor (.-agent localization) impl-localize! object pattern))
+   (if (message-pattern? pattern)
+     (send-without-thread-binding-reset javafx-executor (.-agent localization) impl-localize! object pattern)
+     (send-without-thread-binding-reset javafx-executor (.-agent localization) impl-unlocalize! object))
```

Comment:
The previous code was wrong, actually. The thing is, we can `localize!` either with simple values like strings, or localization-aware values like message patterns. We added a listener only when we localized with message patterns. But in the outline view, the tree cells may be repeatedly localized with both strings and message patterns. So it's possible that when we call `localize!` with a string (which will apply the localization once), the same object was already localized with a message pattern, so changing the language would refresh the label. So, we need to remove a listener if it exists.